test/Dapr.E2E.Test/Workflows/WorkflowTest.cs

```diff
@@ -41,19 +42,63 @@ public async Task TestWorkflows()
             var health = await daprClient.CheckHealthAsync();
             health.Should().Be(true, "DaprClient is not healthy");
 
+            Thread.Sleep(10000);
```

Comment:
Why do we poll for unhealthy above this and then sleep for 10s? Can we not poll for healthy instead?